### ISSUE TITLE
Global types with enum values

### ISSUE THREAD

**Issue by chrbala** [cometkim/gatsby-plugin-typegen#81]
Because of the [update to export only global types](https://github.com/cometkim/gatsby-plugin-typegen/commit/00ec5250fbb5bb924717a6e25173c7b36f981279), enum values break. That is, because enum values pass through the types to the underlying Gatbsy compilation process, the Gatsby compilation process doesn't know how to handle the global types. Example below.

GraphQL definition
```graphql
enum MyEnum {
  SOME_VALUE
  OTHER_VALUE
}
```

This gets transformed to the following in ``gatsby-types.ts``
```typescript
enum MyEnum {
  SOME_VALUE = 'SOME_VALUE',
  OTHER_VALUE = 'OTHER_VALUE',
}
```

Typescript then allows the following usage:
```typescript
const acceptMyEnum = (value: GatsbyTypes.MyEnum) => value == GatsbyTypes.MyEnum.SOME_VALUE ? "something" : "something else";
```

But the actual Gatsby build fails where GatbsyTypes is needed in the runtime because it can't find GatsbyTypes with an error of ``"GatsbyTypes" is not defined``. Is it possible to emit both the global types and the older importable types? Otherwise, is there better solution to this?

**Comment by cometkim** [comment]
Duplicates #78 

These enum values will be changed into an internal type union after #79 is merged.

**Comment by orblazer** [comment]
Hello @cometkim, this bug is still present.
In the release `2.2.0` that PR #80 is missing, probably fail.

**Comment by cometkim** [comment]
Oh no :cry:  @orblazer you right. Thank you for notifying it.

![image](https://user-images.githubusercontent.com/9696352/97787345-ddaced80-1bf4-11eb-9671-81ff7ef46f75.png)

**Comment by cometkim** [comment]
Published at v2.2.2

https://github.com/cometkim/gatsby-plugin-typegen/pull/135
